src/profile.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+
+#include "value.h"
+#include "aggregate/aggregate.h"
+
+#define CLOCKS_PER_MILLISEC  ((__clock_t) 1000)
```

Comment:
Better use `CLOCKS_PER_SEC` (from `time.h`) directly (in `profile.c` and `index.c`).

Reply:
We need better resolution. Most replies are smaller than a millisec.

Reply:
What I mean, CLOCKS_PER_SEC needs to be somehow involved in the calculation, as it may change between platforms.